### PR TITLE
SAIC-105 missing path to import "action"

### PR DESCRIPTION
**Pull request by wzbbbb**
All other files in the directory have path defined:

grep "import action" *

converter.py:import action
copy_var.py:from cloudferrylib.base.action import action
create_reference.py:from cloudferrylib.base.action import action
get_info_iter.py:from cloudferrylib.base.action import action
is_end_iter.py:from cloudferrylib.base.action import action
is_option.py:from cloudferrylib.base.action import action
merge.py:from cloudferrylib.base.action import action
rename_info.py:from cloudferrylib.base.action import action
transporter.py:from cloudferrylib.base.action import action

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/192/

**Comment by roman-verchikov** [comment]
There's few more actually:
```
$ find . -name '*.py' | xargs grep '^import action'
./cloudferrylib/base/__init__.py:import action
./cloudferrylib/base/action/converter.py:import action
./cloudferrylib/os/__init__.py:import actions
```

**Comment by miarmak** [comment]
Actually I am wondering, how it worked before)

**Comment by wzbbbb** [comment]
For other files, I did not need to change them.

**Comment by roman-verchikov** [comment]
>  wzbbbb commented 22 minutes ago
> For other files, I did not need to change them.

??? Let's fix all the occurences of this issue so that they won't appear in future

**Comment by wzbbbb** [comment]
Can not update the other 2 __init__ files. It causes the migration abort.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/194/

File: cloudferrylib/base/__init__.py [hunk 1]
```diff
@@ -1,7 +0,0 @@
-import action
-import compute
-import identity
-import image
-import network
-import resource
-import storage
```
Comment (miarmak) [inline cloudferrylib/base/__init__.py]: No newline here

File: cloudferrylib/os/__init__.py [hunk 2]
```diff
@@ -1 +0,0 @@
-import actions
```
Comment (miarmak) [inline cloudferrylib/os/__init__.py]: And here

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/208/

**Comment by miarmak** [comment]
Thanks! +1 from my side.

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**